digitalstrategie/settings/base.py

```diff
@@ -52,6 +52,7 @@
     'apps.settings',
     'apps.snippets',
     'apps.gruenbuch',
+    'digitalstrategie'
```

Comment:
not sure if this is the best thing to do?

Reply:
I think, I will move it to the app where we have the other footer settings? But that would be the snippets... Or we add an contrib app. Or just move it to the home app as that is the main one. Anyway, it should be in the apps folder and I will have a look where it fits.